airsonic-main/src/main/resources/liquibase/6.5/add-multiple-genres-join.xml

```diff
@@ -0,0 +1,88 @@
+<databaseChangeLog
```

Comment:
I think that the liquibase/6.5 is not correct. I'm not sure but it should be named after the target airsonic version. 
@muff1nman